CONTRIBUTING

```diff
@@ -3,7 +3,8 @@ Scott Berrevoets @sberrevoets (Lyft, secondary) - Until 2/2023
 Patrick Balestra @BalestraPatrick (Spotify) - Until 2/2023
 Dan Fleming @dflems (Spotify, Secondary) - Until 2/2023
 Steven Hepting @stevenhepting (Airbnb) - Until 2/2023
+Francisco Díaz @fdiaz (Airbnb) - Until 2/2023
```

Comment:
You should likely add `, Secondary` in the brackets.

Reply:
Updated! https://github.com/MobileNativeFoundation/foundation/pull/8/commits/e9168011a4de0ba1a0cad4cdc3743aeef841f7c1